game_eggs/steamcmd_servers/ark_survival_ascended/egg-ark--survival-ascended.json

```diff
@@ -154,7 +154,7 @@
             "name": "RCON Port",
             "description": "required for console commands and proper server shutdown (saveworld\/exit), doesn't have to be open to the public",
             "env_variable": "RCON_PORT",
-            "default_value": "37015",
+            "default_value": "27020",
```

Comment:
There is also no reason to change the default port. It's internal only unless you want it exposed outside the container.

Reply:
Doesn't have to be changed, can be any port anyways.
Ark Survival Evolved used 27020 for RCON, again just to make it clearer for people coming from Ark Survival Ascended.